docs/user-guide/deployments.md

```diff
@@ -567,6 +567,7 @@ Kubernetes marks a Deployment as _progressing_ when one of the following tasks i
 * The Deployment is in the process of creating a new ReplicaSet.
 * The Deployment is scaling up an existing ReplicaSet.
 * The Deployment is scaling down an existing ReplicaSet.
+* New pods become available.
```

Comment:
Not necessary. I asked this just because current docs should only mention 1.5 features/behavior.